src/api/spec/models/package_spec.rb

```diff
@@ -41,6 +41,20 @@
     login(user)
   end
 
+  describe 'validations' do
```

Comment:
You're overdoing it. I'm sure validations are tested in the activerecord code base :-)

Reply:
I think testing validations is good to ensure that a change don't break them. We also have it on several other places.

Reply:
What can you change that will break `validates`? The activerecord source code?

Reply:
Not validates but you can apply rules to `validates` as you know. So if someone is modifying this, he can break things accidentally. We even have model validations in our test suite. So I really don't see the point in removing this.

Reply:
If someone is changing this purposefully, that same someone will also change your test. You are not protecting the code base from anything, you are just making it harder to change.